src/client/core/utils/dom.js

```diff
@@ -323,6 +323,35 @@ export function isElementFocusable ($element) {
             $element.css('visibility') !== 'hidden');
 }
 
+export function isIFrameWindowInDOM (win) {
+    //NOTE: In MS Edge, if iframe is removed from DOM browser throws an exception when accessing window.top
+    //and window.frameElement. Fortunately, setTimeout is set to undefined in this case.
+    if (browserUtils.isMSEdge && !win.setTimeout)
```

Comment:
Just curios, can we get rid of browser detection here and just use the feature detection here?